packages/sui-segment-wrapper/package.json

```diff
@@ -5,12 +5,14 @@
   "main": "lib/index.js",
   "scripts": {
     "lib": "sui-js-compiler",
+    "postlib": "npm run set:version",
     "prepublishOnly": "npm run umd && npm run lib",
+    "set:version": "sed -i.bak \"s/process\\.env\\.VERSION/\\\"$npm_package_version\\\"/g\" lib/segmentWrapper.js && rm lib/segmentWrapper.js.bak",
```

Comment:
Sets package version by manipulating the compiled file that uses the `process.env.VERSION`.